### ISSUE TITLE
Compiler for Windows ARM64

### ISSUE THREAD

**Issue by maxgio92** [goreleaser/goreleaser-cross#24]
Hi, first of all thanks for this great work which reduces a lot the burden of cross-compilation with CGO :)

The list of supported platforms in the [`README.md`](https://github.com/goreleaser/goreleaser-cross#supported-toolchainsplatforms) include Windows on ARM64, but it seems that MinGW's clang is not present (`/llvm-mingw/llvm-mingw/bin/aarch64-w64-mingw32-gcc`) in the published container image.

Am I missing something?

Thank you

**Comment by troian** [comment]
should be about there. bear in mind `/llvm-mingw/llvm-mingw/bin/` is not included in the path.
```shell
root@c20e1fcd1089:/# /llvm-mingw/llvm-mingw/bin/aarch64-w64-mingw32-gcc
clang-15: error: no input files
```

**Comment by maxgio92** [comment]
Thanks @troian. Maybe I'm pointing to the wrong needed container image, but for the currently latest v1.19, I'm getting this:

```
docker run  --entrypoint ls goreleaser/goreleaser-cross:v1.19 /llvm-mingw/
ls: cannot access '/llvm-mingw/': No such file or directory
```

**Comment by troian** [comment]
thats weird, thanks for reporting.
you can use `v1.19.2` for now, it is for sure there

**Comment by troian** [comment]
should be all fixed in retagged `v1.19.5`. Path is `/llvm-mingw/bin`

**Comment by maxgio92** [comment]
Thank you @troian!
